games/mohaa/entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+cd /home/container
+
+# Make internal Docker IP address available to processes.
+export INTERNAL_IP=`ip route get 1 | awk '{print $NF;exit}'`
```

Comment:
Please add the new fix from https://github.com/parkervcp/yolks/commit/cd6b94986888030898f126fc1ce1ff977c80a82b